modules/core/src/main/java/org/apache/ignite/internal/processors/datastructures/GridCacheSetProxy.java

```diff
@@ -53,7 +55,7 @@
         };
 
     /** Delegate set. */
-    private GridCacheSetImpl<T> delegate;
+    private GridCacheSet<T> delegate;
```

Comment:
can we revert back GridCacheSetImpl to GridCacheSet ? would it still work? If yes, may be we should do it